config/initializers/new_framework_defaults_7_1.rb

```diff
@@ -1,10 +1,10 @@
 # Be sure to restart your server when you modify this file.
 #
-# This file eases your Rails 7.0 framework defaults upgrade.
+# This file eases your Rails 7.1 framework defaults upgrade.
```

Comment:
Since we're already using all `7.1` defaults per the changes in `config/application.rb`, we don't need this file (and we didn't need the existing 7.0 one either but it snuck in there somehow).  https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#configure-framework-defaults